modules/check_in/app/controllers/check_in/v2/appointments_controller.rb

```diff
@@ -29,9 +59,13 @@ def end_date
       end
 
       def statuses
-        s = params[:statuses]
+        s = permitted_params[:statuses]
```

Comment:
Wondering, if we need to have the `statuses` method?  Are going to filter appointments by `status`? If we are going to support filtering by status, then can we please add rspecs for getting appointments by status?

Reply:
removed in 97d7721